addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -635,10 +641,11 @@ odoo.define('website.s_website_form', function (require) {
                 if (!dependencyIsVisible) {
                     return false;
                 }
-
                 const formData = new FormData(this.$target[0]);
-                const currentValueOfDependency = formData.get(dependencyName);
-                return this._compareTo(comparator, currentValueOfDependency, visibilityCondition, between);
+                if(comparator==='contains' || comparator==='!contains')
```

Comment:
You can replace the double check by includes so you don't have to write twice comparator and should we add a third check it will be easier: ["contains", "!contains"].includes(comparator)